Makefile

```diff
@@ -699,13 +699,11 @@ lint-go:
 	golangci-lint run -c .golangci.yml --build-tags='$(LIBFIDO2_TEST_TAG) $(TOUCHID_TAG) $(PIV_TEST_TAG)' $(GO_LINT_FLAGS)
 
 .PHONY: fix-imports
```

Comment:
Should we default this to the buildbox target and have a separate one that runs gci locally? Akin to `make grpc` and `make grpc/host` ?

Reply:
Changed